lib/task/actor/actorMoveRelatedDescriptionsTask.class.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+class ActorMoveRelatedDescriptionsTask extends arBaseTask
+{
+  protected function configure()
+  {
+    $this->addArguments(array(
+      new sfCommandArgument('source', sfCommandArgument::REQUIRED, 'The slug of the source actor'),
+      new sfCommandArgument('target', sfCommandArgument::REQUIRED, 'The slug of the target actor'),
+    ));
+
+    $this->addOptions(array(
+      new sfCommandOption('application', null, sfCommandOption::PARAMETER_OPTIONAL, 'The application name', true),
+      new sfCommandOption('env', null, sfCommandOption::PARAMETER_REQUIRED, 'The environment', 'cli'),
+      new sfCommandOption('connection', null, sfCommandOption::PARAMETER_REQUIRED, 'The connection name', 'propel'),
+      new sfCommandOption('skip-index', null, sfCommandOption::PARAMETER_NONE, "Skip Elasticsearch indexing"),
+    ));
+
+    $this->namespace = 'actor';
+    $this->name = 'move-related-descriptions';
```

Comment:
Thanks @djjuhasz!

Since we're using the `source` and `target` parameters, I think "move" is better than "update" but I agree that "description-relations" is less confusing, so I went with `move-description-relations` in the end. I also struggle in between "relations" and "relationships" so, please, let me know if you think the later is better.

I'd like to hear @fiver-watson thoughts about this too ...